jobs/generation/Utilities.groovy

```diff
@@ -1274,6 +1274,74 @@ class Utilities {
     }
 
     /**
+     * Creates the standard job view for a given folder. This does not create the folder.
+     *
+     * @param folderName The folder to create a view for.
+     * @param jobName The name of the job, to use in customizing text. Defaults to folderName if not specified.
+     * @param viewName The name to give the standard view. Defaults to 'Official Builds'.
+     * @param filterRegex The regex that determines what jobs should display in the view.
+     * @return The created view
+     */
+    def static registerFolderView(String folderName, String jobName = null, String viewName = 'Official Builds', String filterRegex = /.*(?<!prtest)$/) {
+        jobName = jobName ?: folderName
+        // Create a view for all jobs in this folder that don't end with prtest
+        return dashboardView("${folderName}/${viewName}") {
```

Comment:
This probably won't work without context.  You'll need to pass the factory object ('this' pointer in netci.groovy/MetaGenerator.groovy) along.  If passed to parameter dslFactory:

`dslFactory.dashBoardView`